HACKING.md

```diff
@@ -155,8 +155,14 @@ Use the same user and password that you used to log into your Linux desktop.
 
 http://localhost:9090
 
-After every change to your sources, run `make` to update all the webpacks, and
-reload cockpit in your browser.
+After every change to your sources the webpacks need to be rebuilt: You can
+just run run `make` to update everything that has changed; for iterating
+faster, you can run webpack in "watch" mode on the particular page that you are
+working on, which reduces the build time to less than a third. E. g.
+
+    $ ESLINT=0 tools/webpack-make -d dist/systemd/Makefile.deps -w
```

Comment:
hmm, so I cannot just watch it all? I need to be specific?

Reply:
Yes, both a direct `webpack`, `webpack --watch`, or `tools/webpack-make` without `-d` is utterly broken right now. I have not yet looked into why, or whether this is even supposed to work. For now we only support building them individually. I figure it's also what you want for fast iteration -- you don't want to rebuild everything when you touch a file in pkg/lib/, for example.